syft/pkg/apk_metadata_test.go

```diff
@@ -31,3 +34,45 @@ func TestApkMetadata_pURL(t *testing.T) {
 		})
 	}
 }
+
+func TestApkMetadata_fileOwner(t *testing.T) {
+	tests := []struct {
+		metadata ApkMetadata
+		expected []string
+	}{
+		{
+			metadata: ApkMetadata{
+				Files: []ApkFileRecord{
+					{Path: "/somewhere"},
+					{Path: "/else"},
+				},
+			},
+			expected: []string{
+				"/somewhere",
+				"/else",
+			},
+		},
+		{
+			metadata: ApkMetadata{
+				Files: []ApkFileRecord{
+					{Path: "/somewhere"},
+					{Path: ""},
+				},
+			},
+			expected: []string{
+				"/somewhere",
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(strings.Join(test.expected, ","), func(t *testing.T) {
+			var i interface{}
+			i = test.metadata
+			actual := i.(fileOwner).ownedFiles()
+			for _, d := range deep.Equal(test.expected, actual) {
```

Comment:
Not required — but it could be nice to start trying out Testify's assert package in earnest, in hopes of finding a better way to approach assertions in our tests

Reply:
+1 on normalizing moving forward with testify

Reply:
hadn't realized we reached consensus on using testify, I'll start using it after this PR :)

Reply:
🎉 nice!